contracts/AnchorVault.vy

```diff
@@ -145,11 +185,28 @@ def change_admin(new_admin: address):
 
     Setting the admin to zero ossifies the contract, i.e. makes it irreversibly non-administrable.
     """
-    assert msg.sender == self.admin # dev: unauthorized
+    self._assert_authorized(msg.sender)
+    # we're explicitly allowing zero admin address for ossification
     self.admin = new_admin
     log AdminChanged(new_admin)
 
 
+@external
+def bump_version():
```

Comment:
Is version supposed to be incremented by adding another `finalize_upgrade_vX` or only by means of bumping version?

My first guess was that the version is to be incremented in two ways:
- by bumping if a delegated changed (non-backwards-compatible)
- by `finalize_upgrade_vX` if AnchorVault changed (according LIP-10)

But `bump_version` doc sounds as if only bumping from now on.

I favour my first guess option a bit.

Thus I would add some clarifying note near `version: public(uint256)` or `bump_version`.